### DISCUSSION TITLE
upload-file widget/example

### DISCUSSION THREAD

**Discussion by gsal** [causalens/dara#331] [Q&A] [answered]
Will somebody be kind enough to provide a minimal example on how to setup a widget to upload a file?

**Comment by patricia-causalens** [comment, marked as answer]
Hi @gsal, 

You can use our `Dropzone` component to upload files, here is a small example of how this might be set up:
```python
from dara.components import UploadDropzone

def resolve_file_uploaded(file:bytes, file_name: str):
    print(file_name)

UploadDropzone(
    accept="image/png, image/jpeg",
    resolver=resolve_file_uploaded
)
```

Please check our [Dropzone](https://dara.causalens.com/docs/generated/dara/reference/dara/components/common/dropzone) docs to learn more about the props it accepts, and do let us know if you have any questions about how to use it.
